File: mindvault-ai-python/main/python/models/rerank_source_item.py
from pydantic import BaseModel


class RerankSourceItem(BaseModel):
    type: str # "post" / "user" / "note"
    id: str # 唯一标识，跳转详情页用
    title: str # 标题 / 昵称
    content: str # 正文 / 简介 / 笔记内容

File: mindvault-ai-api/src/main/java/org/example/mindvaultaiapi/dto/RerankSourceItem.java
package org.example.mindvaultaiapi.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class RerankSourceItem {

    private String type;// "post" / "note"
    private String id;// 唯一标识，跳转详情页用
    private String title;// 标题
    private String content;// 正文 / 笔记内容
}
